web-api/src/main/java/com/graphhopper/GHRequest.java

```diff
@@ -37,6 +37,7 @@
 public class GHRequest {
     private List<GHPoint> points;
     private String profile = "";
+    private final int DEFAULT_TOLERANCE = 100; //Default tolerance that can be changed
```

Comment:
Typically such constants should be static. But since this one is not really used anywhere else you could also just write `private int tolerance = 100` below.